source/features/toggle-files-button.tsx

```diff
@@ -25,20 +29,30 @@ function addButton(): void {
 			type="button"
 			className="btn-octicon rgh-toggle-files"
 			aria-label="Toggle files section"
-			aria-expanded="true"
 		>
 			<FoldIcon/>
 			<UnfoldIcon/>
 		</button>
 	);
 }
 
-function init(): void {
-	const repoContent = select('.repository-content')!;
+async function toggleHandler(): Promise<void> {
+	const isHidden = select('.repository-content')!.classList.toggle('rgh-files-hidden');
```

Comment:
```suggestion
	// Todo use `undefined` after https://github.com/fregante/webext-storage-cache/issues/25
	const isHidden = select('.repository-content')!.classList.toggle('rgh-files-hidden');
```

I think we should add a todo so we don't forget about it

Reply:
I already added the condition, there’s no need. 

Also use the same style as we always use: `TODO:`